internal/pkg/manifest/validate_test.go

```diff
@@ -231,8 +232,9 @@ func Test_HandleConnectorApprovalRequired(t *testing.T) {
 }
 
 // Setup
-func setupCommonMocks() (ctx context.Context, clients *shared.ClientFactory, clientsMock *shared.ClientsMock, log *logger.Logger, mockApp types.App, mockAuth types.SlackAuth) {
+func setupCommonMocks(t *testing.T) (ctx context.Context, clients *shared.ClientFactory, clientsMock *shared.ClientsMock, log *logger.Logger, mockApp types.App, mockAuth types.SlackAuth) {
```

Comment:
:books: No blocker but sometimes arguments and returns of this amount on a single line are no fun to parse as the reader.

No need to change it and this is opinion too! Also I've tried using linters to tackle this with linters that `max` a line length but handling cases where this is needed is not always convenient TBH.

Reply:
Yea, that's a fair point @zimeg! If we can use our linter to enforce a max line length before declaring the function as multi-line, then I think that's best!